lib/manageiq/api/common/entitlement.rb

```diff
@@ -0,0 +1,35 @@
+module ManageIQ
+  module API
+    module Common
+      class EntitlementError < StandardError; end
+
+      class Entitlement
+        def initialize(identity)
```

Comment:
@syncrou Can this default to picking up the identity from the Request object and raise an error if its nil.
Is this expected to be called from the base controllers for every micro service?

Reply:
Eventually it will, but until other micro services add the check for entitlements we're allowing everything through unless the entitlements hash exists. If it exists we raise exceptions if it is invalid.

https://projects.engineering.redhat.com/browse/SSP-327